product_docs/docs/biganimal/release/administering_cluster/projects.mdx

```diff
@@ -48,9 +48,11 @@ Before creating a project:
 
 To create a new project:
```

Comment:
can you also add tags to an existing project? do we need docs on how to do that?

Reply:
I have added this to the docs and renamed the title "To assign a tag to an existing project" for better understanding.